application/common/models/Mfa.php

```diff
@@ -267,6 +267,75 @@ public function verify($value, string $rpOrigin = ''): bool
         return false;
     }
 
+
+
+    /**
+     * @param string|array $value
+     * @return bool
+     * @throws ServerErrorHttpException
+     * @throws TooManyRequestsHttpException
+     * @throws \Throwable
+     * @throws \yii\db\StaleObjectException
+     */
+    public function verifyRegistration($value, string $rpOrigin = ''): bool
```

Comment:
What would you think of adding a `$credId` optional parameter on `verify()`, like we did on `delete()`?

Reply:
I'll look into it.  I kind of doubt that it will be adequate, but maybe it will.

Reply:
Where would the credID come from?  Would the UI have to send it in to id-broker just for verifying the registration (and not for login)?
So, in addition to having 
`'POST   mfa/<id:\d+>/verify'                                  => 'mfa/verify'`
we would also have
`'POST   mfa/<id:\d+>/verify<webauthnId:\d+>'     => 'mfa/verify'`

Reply:
Yes, the UI could send it in. Although, as you say, it won't be needed during login, as a major purpose of all of this is for all of their WebAuthn keys to start blinking during login.

Just a thought.

Reply:
Actually, there's no need for an unverified webauthn credential to be kept (as you mentioned elsewhere, I think). So, instead of expecting a webauthnId value, maybe we could have an additional route like 
`'POST mfa/<id:\d+>/verify/<type:^(registration|login)$>' => 'mfa/verify'`

How does that sound?

Reply:
Yeah, that could work. As long as we try to reduce the amount of duplicated logic (where feasible), I think your idea is a good approach. 👍